unit/src/ParsedFunctionTest.C

```diff
@@ -34,10 +34,12 @@ ParsedFunctionTest::init()
 
   InputParameters mesh_params = _factory->getValidParams("GeneratedMesh");
   mesh_params.set<MooseEnum>("dim") = "3";
+  mesh_params.set<std::string>("name") = "mesh";
```

Comment:
Because the Action system generates the name not the factory.

On Wed, Feb 4, 2015 at 4:23 PM, Cody Permann <notifications@github.com>
wrote:

> In unit/src/ParsedFunctionTest.C
> <https://github.com/idaholab/moose/pull/4642#discussion_r24130180>:
>
> > @@ -34,10 +34,12 @@ ParsedFunctionTest::init()
> >
> >    InputParameters mesh_params = _factory->getValidParams("GeneratedMesh");
> >    mesh_params.set<MooseEnum>("dim") = "3";
> > +  mesh_params.set<std::string>("name") = "mesh";
>
> So then why set it here in the unit tests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/4642/files#r24130180>.
>